File: cs2100/Lecture/COde/InputOutputV2.c
// InputOutputV2.c
// Demonstrate scanf() and printf() functions
#include <stdio.h>

int main(void) {
	int age;
	double cap;

	printf("What are your age and CAP? ");
	scanf("%d %lf", &age, &cap);
	printf("You are %d years old, and your CAP is %f\n", age, cap);

	return 0;
}

File: cs2100/Lecture/COde/InputOutputV2.py
def main():
  user_input = input("What are your age and CAP? ")
  age,cap = user_input.split(" ")
  age,cap = int(age),float(cap)
  print("You are", age, "years old, and your CAP is", cap)

  return 0

if __name__ == "__main__":
  main()
